src/50_Samples_%26_Templates/Product.html

```diff
@@ -260,13 +260,13 @@
       <!-- ## Navigation -->
       <!-- Use the [amp-sidebar](/components/amp-sidebar/) component to give customers the chance to quickly jump to any of your product categories.-->
       <amp-sidebar id="drawermenu" layout="nodisplay">
-        <div class="topheader">
-          <a href="/" class="home">Products by Example</a>
+        <div class="topheader m1">
```

Comment:
is topheader still needed? I think we can get rid of the div and add "block p1" to the link instead.